packages/entities/entities-shared/src/components/entity-filter/EntityFilter.vue

```diff
@@ -35,92 +35,99 @@
         class="kong-ui-entity-filter-backdrop"
         @click="toggleMenu"
       />
-      <KMenu
+      <div
         v-show="showMenu"
         class="kong-ui-entity-filter-menu"
       >
-        <template #body>
-          <KMenuItem
-            v-for="(field, index) in searchableFields"
-            :key="field.value"
-            :data-testid="field.value"
-            expandable
-            :last-menu-item="index === searchableFields.length - 1"
+        <div
+          v-for="field in searchableFields"
+          :key="field.value"
+          class="kong-ui-entity-filter-menu-item"
+          :data-testid="field.value"
+        >
+          <span
```

Comment:
Can you add `role="button"` and `tabindex="0"` to these expand/collapse spans please?

Reply:
Good call👍. Updated.